bodhi/tests/server/services/test_updates.py

```diff
@@ -2238,6 +2429,8 @@ def test_request_to_stable_based_on_stable_karma(self, *args):
         up = Update.get(nvr, self.db)
         up.status = UpdateStatus.testing
         up.request = None
+        eq_(len(up.builds), 1)
```

Comment:
I added this line so that it would fail there rather than at the next line which does ``up.builds[0]``, if there are more than one build, they should all be adjusted or the behaviour may differ.